R/chk_namespace.R

```diff
@@ -8,8 +8,8 @@ CHECKS$no_import_package_as_a_whole <- make_check(
   preps = "namespace",
 
   gp = 'not import packages as a whole, as this can cause name
-        clashes between the imported packages. Instead, import
-        only the specific functions you need.',
+        clashes between the imported packages, especially over time.
```

Comment:
```suggestion
        clashes between the imported packages, especially over time as packages change.
```
I think this is an important clarification, but "especially over time" along confused me at first. I think this helps clarify?